<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formProfileManager</class>
 <widget class="QMainWindow" name="formProfileManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>409</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Account Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>350</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>350</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeleteAll">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>350</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete All</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbAccount">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>9</y>
      <width>901</width>
      <height>331</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>7</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Site</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>User Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Password</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Proxy</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Profile</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
